web/src/beta/features/PluginPlayground/Plugins/presets/custom/myPlugin.ts

```diff
@@ -17,6 +17,12 @@ extensions:
         zone: outer
         section: left
         area: top
+  - id: demo-first-infobox
+    type: infoboxBlock
+    name: Demo First Infobox
+  - id: demo-second-infobox
+    type: infoboxBlock
+    name: Demo Second Infobox
```

Comment:
```suggestion
  - id: demo-infobox-block-1
    type: infoboxBlock
    name: Demo Infobox Block 1
  - id: demo-infobox-block-2
    type: infoboxBlock
    name: Demo Infobox Block 2
```
The naming was a little bit misleading, as an extension it should be a `block` (infoboxBlock) not an `infobox`, please update related names as well.

Reply:
done